### ISSUE TITLE
input value will disappear by calling fragments

### ISSUE THREAD

**Issue by rzfang** [riot/ssr#35]
### Environment

Linux Mint 20.3 Cinnamon
node@14.19.1
npm@6.14.16
@riotjs/ssr@8.0.0

### Steps to reproduce the problem.

Can not find any platform which can run ssr, so I paste code here.

file - test.riot.js
```js
export default {
  css: null,
  exports: null,

  template: (
    template,
    expressionTypes,
    bindingTypes,
    getComponent
  ) => template(
    '<input type="button" value="GJ"/>',
    []
  ),

  name: 'test'
};
```

file - test.js
```js
import ssr from '@riotjs/ssr';
import test from './test.riot.js';

const { html: HTML, css: CSS } = ssr.fragments('test', test, {});

console.log(HTML);
```

test.js output - the value becomes empty.
```
<test><input type="button" value=""></test>
```

**Comment by rzfang** [comment]
It looks like that input[type=radio] has the same issue:
`<input type='radio' value='test'/>` becomes `<input type='radio' value=''/>`

BUT, this is workable:
`<input type='radio' value={'test'}/>`

**Comment by GianlucaGuarini** [comment]
I will need to investigate it. For now you can simply use `<input value={'your-value'}/>`. That should work.

**Comment by GianlucaGuarini** [comment]
Fixed in @riotjs/ssr@8.0.1 thank you for reporting it

**Comment by rzfang** [comment]
Verified, the issue gone.
Thanks.
